modules/mailer/mail.go

```diff
@@ -126,11 +126,12 @@ func SendIssueNotifyMail(u, owner *models.User, repo *models.Repository, issue *
 		return tos, nil
 	}
 
+	from := fmt.Sprintf("\"%s\" <%s>", u.DisplayName(), setting.MailService.From)
```

Comment:
`setting.MailService.From` is already in format `"name" <email>` https://github.com/gogits/gogs/blob/master/conf/app.ini#L184